ah-server/src/index.js

```diff
@@ -1,11 +1,19 @@
-import http from 'http';
 import Food from './database/db-manager';
+import express from 'express';
 
-http.createServer((req, res) => {
-  let food = new Food('Neko');
+let app = express();
+
+app.get('/', (req, res) => {
+	let food = new Food('Neko');
   res.writeHead(200, {'Content-Type': 'text/plain'});
   food.print();
   res.end('Hello World!!' + food.toString());
-}).listen(3001, ()=>{
-  console.log('Server running at http://127.0.0.1:3001');
-});
\ No newline at end of file
+})
+
+app.get('/holaAriel', (req,res) => {
```

Comment:
@ArielYanarico please change this "/holaAriel" by "helloWorld" the name of REST-API should not content own names